qiskit/circuit/parameterexpression.py

```diff
@@ -27,8 +27,6 @@
 class ParameterExpression:
     """ParameterExpression class to enable creating expressions of Parameters."""
 
-    __slots__ = ['_parameter_symbols', '_parameters', '_symbol_expr', '_names']
```

Comment:
`__slots__` was recently added to `ParameterExpression`  ( #5028 ). Is is necessary to remove it? Can we add `'_subs_once'` and `'_lambda'` to `__slots__` instead?